pkg/scheduler/apis/config/testing/compatibility_test.go

```diff
@@ -1379,6 +1379,7 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
 
 			sched, err := scheduler.New(
 				client,
+				nil,
```

Comment:
can we make it an option instead?

Reply:
sg, so that the logic that doesn't care about this can stay unchanged.